test/basic_parser.cpp

```diff
@@ -778,6 +778,41 @@ class basic_parser_test
         BOOST_TEST(! validate(js));
     }
 
+    void
+    testIssue113()
+    {
+        string_view s = 
+            "\"\\r\\n section id='description'>\\r\\nAll        mbers form the uncountable set "
```

Comment:
If this is from the issue can we please use a raw string so we don't have to have all these escapes?

Reply:
I used escapes to ensure that the strings are identical (i.e. no extra newlines)

Reply:
I used escapes to ensure that the strings are identical (i.e. no extra newlines)